packages/cozy-stack-client/src/DocumentCollection.spec.js

```diff
@@ -468,6 +468,22 @@ describe('DocumentCollection', () => {
     })
   })
 
+  describe('get', () => {
+    afterEach(() => {
+      jest.resetAllMocks()
+    })
+
+    it('should throw', async () => {
+      jest.spyOn(client, 'fetchJSON').mockImplementation(() => {
+        throw new Error('Not a 404 error')
+      })
+      const collection = new DocumentCollection('io.cozy.todos', client)
+      await expect(collection.get('whatever_id')).rejects.toEqual(
+        new Error('Not a 404 error')
+      )
+    })
```

Comment:
Can we have a test for the case where it is a not_found error. Here since there is change with the `null` I think we expect to get `{ data: null }` but I think we will have `{ data: undefined }` since the result of `dontThrowNotFoundError(error, null)` is not used.